index.js

```diff
@@ -57,7 +64,9 @@ function filter(array, callback) {
 function map(array, callback) {
   const result = [];
```

Comment:
Try again - think about if this is 'filtering' the array or creating a brand new array